sql/updates/world/3.3.5/Abduction.sql

```diff
@@ -0,0 +1,21 @@
+-- *** Quest 11590 "Abduction" ***
+
+-- Condition for source Spell condition type Object entry guid
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=17 AND `SourceGroup`=0 AND `SourceEntry`=45611 AND `SourceId`=0;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(17, 0, 45611, 0, 0, 31, 1, 3, 25316, 0, 0, 12, 0, '', 'Spell Arcane Chains will hit the explicit target of the spell if target is unit Beryl Sorcerer.'),
+(17, 0, 45611, 0, 0, 38, 1, 25, 4, 0, 0, 12, 0, '', 'Spell Arcane Chains will hit the explicit target of the spell if target health percentage must be equal or lower than 25% of max Health.');
+
+DELETE FROM `spell_script_names` WHERE `spell_id`= 45625;
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(45625, 'spell_arcane_chains_character_force_cast');
+
+-- Update creatures
+UPDATE `creature_template` SET `minlevel`=69, `maxlevel`=70, `unit_flags`=32768, `unit_class`=8, `ScriptName`='npc_captured_beryl_sorcerer' WHERE `entry`=25474;
```

Comment:
He takes(or should take) level from player, in DB levels are correct
```txt
    Id  CreatureId  FactionTemplateId  Expansion  MinLevel  MaxLevel     Flags1  Flags2  Flags3  Flags4  Flags5  
------  ----------  -----------------  ---------  --------  --------  ---------  ------  ------  ------  --------
 24118       25474                 35          2        71        72  268959750       0       0       0         0

Reply:
[2] Level: 70 -- from sniff

Reply:
[2] ClassId: 2 -- from sniff so I got that wrong.

Reply:
So I should inform you that, unless they have been updated, all exp, class, and level values where put there by me an long while back over a week of 6 hours a day gathering data from mostly wowhead. So never assume that they are all 100% especially npcs that are summoned for a quest.

Reply:
This creature takes level from player after summon, so level in sniff or wowhead is wrong, SQL row in my comment above is basically a piece of Blizz DB, same values are used currently in TC and doesn't require changes

Reply:
I was 71 when I did this quest and the npc was 70.

Reply:
80 and 80 in 335 sniff
Checked a video from MoP, he changes level to player's (90 > 90). That is weird they changed level in DB at some point, maybe even in WotLK. Better to keep old values to reduce mismatches with db2